src/alchemlyb/convergence/convergence.py

```diff
@@ -82,7 +82,7 @@ def forward_backward_convergence(df_list, estimator="MBAR", num=10, **kwargs):
         raise ValueError(msg)
     else:
         # select estimator class by name
-        estimator_fit = estimators_dispatch[estimator](**kwargs).fit
```

Comment:
I'd rename `estimator_fit` to `my_estimator` — that will make `my_estimator.fit()` a lot nicer to read.

(Unfortunately, `estimator` is already a variable.)